src/ConnectedMode/Migration/Wizard/MigrationWizardController.cs

```diff
@@ -47,30 +47,40 @@ internal class MigrationWizardController : IMigrationWizardController
 
         private readonly IConnectedModeMigration connectedModeMigration;
         private readonly IBrowserService browserService;
+        private readonly IOutputWindowService outputWindowService;
         private readonly ILogger logger;
 
         [ImportingConstructor]
         public MigrationWizardController(IConnectedModeMigration connectedModeMigration,
             IBrowserService browserService,
+            IOutputWindowService outputWindowService,
             ILogger logger)
         {
             this.connectedModeMigration = connectedModeMigration;
             this.browserService = browserService;
+            this.outputWindowService = outputWindowService;
             this.logger = logger;
         }
 
         public void StartMigrationWizard(BoundSonarQubeProject oldBinding)
         {
-            var migrationWizardWindow = new MigrationWizardWindow(oldBinding, connectedModeMigration, OnShowHelp, logger);
+            var migrationWizardWindow = new MigrationWizardWindow(oldBinding, connectedModeMigration, OnShowHelp, OnShowTfvcHelp, logger);
 
             var finishedSuccessfully = migrationWizardWindow.ShowModal();
 
             if (finishedSuccessfully != null && finishedSuccessfully.Value)
             {
                 MigrationWizardFinished?.Invoke(this, EventArgs.Empty);
             }
+            else
+            {
+                // Show the output window in event of an unsuccessful migration
+                outputWindowService.Show();
```

Comment:
This is one of the changes: show the output window pane if migration fails.
Fit and finish i.e. nice to have but not essential